CHANGELOG.md

```diff
@@ -1,4 +1,26 @@
 ## 3.29.0 (Unreleased)
+NOTES:
+* added the `https://www.googleapis.com/auth/cloud-identity` scope to the provider by default ([#6681](https://github.com/terraform-providers/terraform-provider-google/pull/6681))
+* `google_app_engine_*_version`'s `service` field is required; previously it would have passed validation but failed on apply if it were absent. ([#6720](https://github.com/terraform-providers/terraform-provider-google/pull/6720))
+
+FEATURES:
+* **New Data Source:** `google_cloud_identity_group_memberships` ([#6712](https://github.com/terraform-providers/terraform-provider-google/pull/6712))
+* **New Data Source:** `google_cloud_identity_groups` ([#6712](https://github.com/terraform-providers/terraform-provider-google/pull/6712))
```

Comment:
Sorry, I must have forgot to put beta-only on the data source's changelog entries, but the two cloud_identity data sources should just be beta-only.

Reply:
Thanks, done!